README-HELPERS.md

```diff
@@ -292,11 +292,45 @@ $helper->scripts->addCond(
     25                  // (int) optional priority order (default 100)
 ));
 
+// add an internal script
+$helper->scripts()->addInternal(
+    'alert("foo");',     // (string) script
+    1000                // (int) optional priority order (default 100)
+);
+
+// add an internal conditional script
+$helper->scripts()->addCondInternal(
+    'ie6',                // (string) the condition
+    'alert("ie6");',     // (string) script
+    1000                // (int) optional priority order (default 100)
+);
+
+// capture an internal script
+$helper->scripts()->beginInternal(
+    1000                // (int) optional priority order (default 100)
+);
+echo 'alert("captured")';
+$helper->scripts()->endInternal();
+
+// capture an internal conditional script
+$helper->scripts()->beginCondInternal(
+    'ie6',                    // (string) the condition
+    'alert("capture ie6");',  // (string) script
+    1000                      // (int) optional priority order (default 100)
+);
+echo 'alert("capture ie6")';
```

Comment:
When you have passed `alert("capture ie6")` in the `beginCondInternal` and outside, does it only show once the `alert("capture ie6")` or twice ?

I am looking at the output : https://github.com/auraphp/Aura.Html/pull/39/files#diff-8e9751f1c11060391e7a9bb1681dd4f5R332